src/js/components/EditsTable.jsx

```diff
@@ -52,8 +50,17 @@ export const renderTableCaption = (edit, rowObj, pagination) => {
   const editText = length === 1 ? 'edit' : 'edits'
   const captionHeader = `${length} ${name} ${editText} found.`
 
+  if(type === 'macro') {
+    return (
+      <div className="caption">
+        <h3>{captionHeader}</h3>
+        {description ? <p className="usa-font-lead">{description}</p>:null}
+      </div>
+    )
+  }
+
   return (
-    <caption>
+    <caption className="caption">
```

Comment:
Not strictly needed, but I felt like it would be wise to include a way to target the caption whether it was a true caption or just a div.